### PR TITLE
Status bar

### PR DESCRIPTION
**Pull request by SamBrownMT**

### CODE REVIEW

File: backoffice/src/wwwroot/scss/site.scss [hunk 1]
```diff
@@ -56,3 +56,17 @@ div.form-sections {
...
+    .progress-bar {
+        border-right: 1px solid #000;
+    }
+
+    :last-child {
+        border-right: none;
+    }
+}
+
+.button-text {
```
Comment (slkendell-mt) [inline backoffice/src/wwwroot/scss/site.scss:70]: Hmmm, not sure about this one. I think wrap is fine in certain situations and we don't want this absolutely everywhere (esp for mobile views etc)

If it's a specific case where you absolutely want it in all cases, in scss you can wrap it in a context

.thisDiv{
   .buttonTextInHere{
     some: style;
   }
}

Comment (slkendell-mt) [reply backoffice/src/wwwroot/scss/site.scss:70]: Preferably, you can also put the bootstrap class for this straight on the button you don't want to wrap:

`<p class="text-nowrap">No wrap text.</p>`

Comment (SamBrownMT) [reply backoffice/src/wwwroot/scss/site.scss:70]: Having a play around with this and I don't think the .button-text class is necessary

Comment (slkendell-mt) [reply backoffice/src/wwwroot/scss/site.scss:70]: Bin it off

**Review by slkendell-mt — APPROVED** [review]

**Pull request merged**